### PR TITLE
chore: Update text message for git remote url placeholder

### PR DESCRIPTION
**Pull request by AnaghHegde**
## Description

> Update text message for git remote url placeholder.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Locally

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/ce/constants/messages.ts [hunk 1]
```diff
@@ -137,37 +129,19 @@ export const INVITE_USERS_ROLE_SELECT_LABEL = () => `Role`;
...
-export const CREATE_PASSWORD_PAGE_PASSWORD_INPUT_LABEL = () => `New password`;
-export const CREATE_PASSWORD_PAGE_PASSWORD_INPUT_PLACEHOLDER = () =>
-  `New Password`;
-export const CREATE_PASSWORD_LOGIN_LINK_TEXT = () =>
-  `Already know the password? Login`;
-export const CREATE_PASSWORD_PAGE_TITLE = () => `Set password`;
-export const CREATE_PASSWORD_SUBMIT_BUTTON_TEXT = () => `Create`;
-export const CREATE_PASSWORD_PAGE_SUBTITLE = () =>
-  `Set a new password for your account `;
-
```
Comment (IAmAnubhavSaini) [inline app/client/src/ce/constants/messages.ts:158]: These are removed. Are we sure that we don't need these?

**Review by IAmAnubhavSaini — APPROVED** [review]

File: app/client/src/ce/constants/messages.ts (hunk 1 above)
Comment (mohanarpit) [reply app/client/src/ce/constants/messages.ts:158]: I didn't find these constants anywhere in the code base. Hence, I cleaned it up.

**Pull request merged**